cmd/main.go

```diff
@@ -43,33 +44,34 @@ func main() {
 	isArgocdCLI := false
 
 	switch binaryName {
-	case "argocd", "argocd-linux-amd64", "argocd-darwin-amd64", "argocd-windows-amd64.exe":
+	case common.CLI:
```

Comment:
```suggestion
	case common.CLI:
```
I think this should be changed to what it was. With this change, we are only referencing to `argocd` and not other binaries built, such as the one built on windows etc.